packages/fuels/src/cli/commands/init/index.ts

```diff
@@ -1,21 +1,24 @@
-import { type Command } from 'commander';
+import type { Command, Option } from 'commander';
 import { existsSync, writeFileSync } from 'fs';
 import { globSync } from 'glob';
 import { join, relative, resolve } from 'path';
 
 import { renderFuelsConfigTemplate } from '../../templates/fuels.config';
 import { log } from '../../utils/logger';
 
+interface CommandWithInternals extends Command {
+  options: (Option & { conflictsWith: string[] })[];
```

Comment:
I created this interface to be pragmatic and just take out what is needed without having to introduce a structural refactoring to get the `Option`s and their flags that are used [below](https://github.com/FuelLabs/fuels-ts/pull/2558/files#diff-1d2833f29f77c976e789bbe891b340cf95b11435b775736098d1887ed7d6ac14R53). If the property ever becomes private, our tests will fail and we can then do a refactor.